test/intern/mixins/Tree.js

```diff
@@ -135,15 +139,62 @@ define([
 	}
 
 	test.suite('Tree', function () {
-		test.suite('large family expansion', function () {
 
-			test.beforeEach(function () {
-				createGrid();
+		function makeBeforeEach(setStoreAfterStartup) {
+			return function () {
+				createGrid({}, setStoreAfterStartup);
 
 				// Firefox in particular seems to skip transitions sometimes
 				// if we don't wait a bit after creating and placing the grid
 				return wait();
+			};
+		}
+
+		test.suite('no store', function () {
+
+			test.afterEach(destroyGrid);
+
+			test.test('call expand', function () {
+				grid = new (declare([OnDemandGrid, Tree]))(
+					{
+						columns: [
+							{ renderExpando: true, label: 'id', field: 'id' },
+							{ label: 'value', field: 'value' }
+						]
+					}
+				);
+				document.body.appendChild(grid.domNode);
+				grid.startup();
+
+				grid.expand(0);
```

Comment:
While I was reviewing the expand method, I saw that target was being tested before the actual expansion was done.

`if (target && target.mayHaveChildren && (noTransition || expand !== isExpanded)) {`

But not before that.  So I added the guard and this test.  Without the guard, this test throws an error and the test fails.  

Before, you could pass a row id for a row that exists but did not contain an expando icon and expand would do nothing and throw no error but if you passed an id for a row that doesn't exist, it would blow up.  I tried to make the two scenarios more consistent.